packages/apps/plugins/plugin-help/src/components/Shortcuts/ShortcutsList.tsx

```diff
@@ -20,11 +21,16 @@ export const ShortcutsList = () => {
   });
 
   return (
-    <dl className='is-fit grid grid-cols-[min-content_minmax(12rem,1fr)] gap-2 mlb-4'>
+    <dl
+      className={mx(
+        'is-fit grid grid-cols-[min-content_minmax(12rem,1fr)] gap-2 mlb-4',
+        'text-neutral-700 dark:text-neutral-300',
```

Comment:
Prefer to use semantic tokens for this kind of thing — should we mint a semantic token for this value? What is the purpose of these colors?